### ISSUE TITLE
Automatically create test issues for components that violate accessibility standards

### ISSUE THREAD

**Issue by hirotomoyamada** [yamada-ui/yamada-ui#5322]
### Description

Automatically create test issues for components that violate accessibility standards.

In v1, it was implemented in [a11y-issues.ts](https://github.com/yamada-ui/yamada-ui/blob/v1/scripts/a11y-issues.ts).

### Problem Statement/Justification

It is important to automatically create issues for components that violate accessibility standards to maintain health.

### Are you willing to participate in implementing this feature and create a pull request include the implementation?

None
